### PR TITLE
Clarify pull request usage in README

### PR DESCRIPTION
**Pull request by tavaresrodrigo**
This pull request addresses an issue in the policy-collection README where the usage of pull requests may be confusing.

The current text suggests using a pull request to pull changes, which is not accurate. A pull request should be used to propose merging changes.

**Changes Made:**
- Updated the sentence from:
  ```
  Fork this repository and use the forked version as the target to run the sync against. This is to
  avoid unintended changes to be applied to your cluster automatically. To get latest policies from
  the `policy-collection` repository, you can pull the latest changes from `policy-collection` to your
  own repository through a pull request. Any further changes to your repository are automatically be
  applied to your cluster.
  ```
  to:
  ```
  Fork this repository and use the forked version as the target to run the sync. This avoids unintended changes to your cluster. 
  To get the latest policies from the `policy-collection` repository, pull the latest changes from `policy-collection` and then create a pull request to merge these changes into your forked repository. Any further changes to your repository will be applied to your cluster automatically.
  ```

This change ensures that the instructions are clear and technically accurate, helping users understand the correct use of pull requests.

Signed-off-by: Rodrigo Tavares    <rotavares@redhat.com>

### CODE REVIEW

**Comment by tavaresrodrigo** [comment]
Signed-off-by: Rodrigo Tavares <[rotavares@redhat.com](mailto:rotavares@redhat.com)>

**Comment by dhaiducek** [comment]
@tavaresrodrigo Thanks for the PR! For the DCO signoff, since there's only one commit you can run these commands to add the signoff:

```bash
git commit --amend --signoff --no-edit
git push --force
```

File: README.md [hunk 1]
```diff
@@ -28,11 +28,8 @@ This is known as `PolicySets` and these contributions are made in directories or
 
 ## Using GitOps to deploy policies to a cluster
 
-Fork this repository and use the forked version as the target to run the sync against. This is to
-avoid unintended changes to be applied to your cluster automatically. To get latest policies from
-the `policy-collection` repository, you can pull the latest changes from `policy-collection` to your
-own repository through a pull request. Any further changes to your repository are automatically be
-applied to your cluster.
+Fork this repository and use the forked version as the target to run the sync. This avoids unintended changes to your cluster. 
+To get the latest policies from the `policy-collection` repository, pull the latest changes from `policy-collection` and then create a pull request to merge these changes into your forked repository. Any further changes to your repository will be applied to your cluster automatically.
```
Comment (dhaiducek) [inline README.md]: Reflow to match the Markdown formatting:

```suggestion
Fork this repository and use the forked version as the target to run the sync. This avoids
unintended changes to your cluster. To get the latest policies from the `policy-collection`
repository, pull the latest changes from `policy-collection` and then create a pull request to merge
these changes into your forked repository. Any further changes to your repository will be applied to
your cluster automatically.
```

**Review by dhaiducek — CHANGES_REQUESTED** [review]
Thanks for the update! I think the wording looks good--just needs some linting and it'll be good to go!

**Comment by dhaiducek** [comment]
We prefer to minimize the number of commits--would you please squash these commits into a single commit? (Let me know if any guidance is needed on squashing.) After that, it should be good to merge.

**Comment by tavaresrodrigo** [comment]
Thanks for the helpful guidance @dhaiducek ! I have squashed all commits.

**Review by dhaiducek — APPROVED** [review]
Thanks for the update, @tavaresrodrigo!

**Pull request merged**